charts/kube-prometheus-stack/Chart.yaml

```
@@ -23,7 +23,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 62.3.0
+version: 62.3.1
```

Hi @fernandezcuesta, can you please bump minor for feature instead?
```suggestion
version: 62.4.0
```